typed-racket-lib/typed-racket/base-env/base-special-env.rkt

```diff
@@ -387,4 +387,9 @@
   [(make-template-identifier 'start-place 'racket/place)
    (-> -Symbol -Module-Path -Symbol (-opt -Input-Port) (-opt -Output-Port) (-opt -Output-Port) -Place)]
    [(make-template-identifier '-mlist 'compatibility/mlist)
-    (-poly (a) (->* (list) a (-mlst a)))])
+    (-poly (a) (->* (list) a (-mlst a)))]
+  ;; for `match`
+  ;; pregexp-matcher
+  [(make-template-identifier 'pregexp-matcher 'racket/match/runtime)
+   (-> (Un -PRegexp -String) -Symbol (-> (Un -String -Bytes) -Boolean))]
```

Comment:
This return type looks wrong; `regexp-match` and thus `pregexp-matcher` can produce more than just booleans.

Reply:
An update to the type was pushed.